src/coreclr/src/scripts/check-definitions.py

```diff
@@ -1,163 +0,0 @@
-#!/usr/bin/env python
```

Comment:
The definition consistency checking never really worked. It’s had warnings for months and no one could see them since we moved it into Arcade. It also doesn’t work with CMake Generator Expressions. @janvorli said it was fine if we removed it.